content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -100,25 +102,36 @@ The following sysctls are known to be _namespaced_:
 - `fs.mqueue.*`,
 - `net.*`.
 
-Sysctls which are not namespaced are called _node-level_ and must be set
-manually by the cluster admin, either by means of the underlying Linux
-distribution of the nodes (e.g. via `/etc/sysctls.conf`) or using a DaemonSet
-with privileged containers.
+Sysctls with no namespace are called _node-level_ sysctls. If you need to set
```

Comment:
why is this text changed? What is an "underlying node"?

Reply:
Even with the change about it still make sense to me. Though, if it is still generally unclear, I can undo the change. @sttts WDYT?

Reply:
I think I introduced this change to try to make it clear that you'd be messing with the underlying OS of the node, outside the scope of Kubernetes.

Reply:
My point is that a node is a fixed concept in kube. What should be the underlying node be? Write "underlying operation system" or "underlying machine". 

Also the "operation system" is misleading. The operation system is visible all the time, to each container via syscalls of the kernel. The point here is that the user-level tooling by the distribution has to be used, not the operation system as the kernel visible to containers.

Reply:
Thanks, that helps. Let me try to fix this better.